lighthouse-core/report/html/report-styles.css

```diff
@@ -1425,12 +1425,12 @@
   text-decoration: underline dotted #999;
 }
 
-.lh-sub-rows:not(:first-child) .lh-sub-row {
```

Comment:
i kinda miss the previous styling.
i wonder if there's a way we can still have zebra striping on the non-subrows and the subrows feel like they belong.

... trying a few things out it seems impossible without way too much time. ah well.

Reply:
> i kinda miss the previous styling.

I also prefer the previous styling. It's simple if the rendering code adds an "even" or "odd" class to all rows generated from a single data value. Then we just need to duplicate the CSS for reach even / odd.

Reply:
> It's simple if the rendering code adds an "even" or "odd" class to all rows generated from a single data value.

true!  i don't mind that.

Reply:
done